### DISCUSSION TITLE
Stat Hider

### DISCUSSION THREAD

**Discussion by Tylerjitp** [runelite/runelite#15680] [Ideas]
Hello all!

For restricted accounts it would be awesome to have a plugin to hide stats on the stats screen in game. Like If I was 10hp I could hide the entire hitpoint box. Also we could hide every other stat besides fish/cook if that was the account build. Would love to know how hard this would be to make or other ideas for this. Thanks!

**Comment by robrichardson13** [comment]
Are you thinking this would just hide the unwanted skills and keep the rest in place, or would it re-shuffle/organize the shown ones to the top?

**Reply by Tylerjitp** [reply]
Just hiding the skills would be awesome. No need to get all fancy with it unless it was easy to do. My ocd drives me crazy and I know a ton of people who feel the same way lmao. Like when you have transparent invy on so that when you hide certain stats they just become transparent or something would even be awesome!

**Reply by robrichardson13** [reply]
I started playing around with this, here's what it could look like if you completely hide specific skills entirely (example of only showing Fishing/Cooking).

Transparent side panel
![image](https://user-images.githubusercontent.com/9492530/191144019-10e4c9d4-2744-4157-acb7-603800c2b90c.png)

Normal side panel
![image](https://user-images.githubusercontent.com/9492530/191144055-1e937a52-74dc-4fb5-b5a1-c29fafb9aaa9.png)

**Reply by Tylerjitp** [reply]
That's incredible! Exactly what I had in mind. How hard is it to be able to move skills? Say put fishing in herblores spot and cook in thief so it's centered? If it is too much of a pain don't bother I'm excited for what you already have shown lol. Thanks for the work on this!

**Comment by BuzzfeedBlue** [comment]
this is an awesome idea for restricted accounts or even just to focus on what you are currently training

**Comment by robrichardson13** [comment]
I just finished v1 of "Skills Organizer" and requested to get it into plugin hub. It allows you to set any skill (including the total level) to be hidden or any other position in the panel.

Here is the PR: https://github.com/runelite/plugin-hub/pull/3365
Here is the source code of the plugin: https://github.com/robrichardson13/skills-organizer

![Skills](https://i.imgur.com/GyKd75i.png)
![Skills Transparent](https://i.imgur.com/sVLsyW8.png)

**Reply by Tylerjitp** [reply]
Thanks so much for your effort on this it's perfect! How long do these typically take to get added?

**Reply by robrichardson13** [reply]
Its live now! Just search "Skills Organizer"
